desktop/core/src/desktop/lib/raz/clients_test.py

```diff
@@ -58,3 +63,39 @@ def test_check_acccess_s3_list_file(self):
 
 
   def test_check_acccess_s3_list_file_no_access(self): pass
+
+class AdlsRazClientTest(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    if not RAZ.IS_ENABLED.get():
```

Comment:
How come we skip those? (should be ran all the time)

Reply:
Will remove for both ADLS and S3.